### ISSUE TITLE
Log exceptions at a higher log level that "Warning"

### ISSUE THREAD

**Issue by richardTingle** [nifty-gui/nifty-gui#445]
Because Nifty produces a lot of logs at the warning level, e.g. 

WARNING: Style: nifty-tab-button-active#text was already registered. The new definition will override the previous.
Nov 07, 2017 9:31:00 PM de.lessvoid.nifty.Nifty registerStyle
WARNING: Style: nifty-treebox-item was already registered. The new definition will override the previous.
Nov 07, 2017 9:31:00 PM de.lessvoid.nifty.Nifty registerStyle

I turned the logging to log only at the SEVERE level, assuming any real problems would be at that level. But even exceptions are raised at the warning level. e.g. this is logged when I deliberately throw a RuntimeException in a ScreenController method called from a button

WARNING: RuntimeException in callMethod(public void signalrouting.blocks.procedure.ProcedureBlockUiController.moveUp(int)) for [signalrouting.blocks.procedure.ProcedureBlockUiController@600c1da7]

Could exceptions, missing methods, missing controllers and things which will full stop prevent the UI from working correctly be raised at a higher level from things like reregistering styles? (I'm assuming reregistering styles doesn't really matter?)

**Comment by richardTingle** [comment]
I managed to do a temporary workaround like this

        Logger.getLogger("de.lessvoid.nifty.Nifty").setFilter(new Filter(){
            @Override
            public boolean isLoggable(LogRecord lr) {
                boolean isReregisterMessage = lr.getMessage().contains("The new definition will override the previous.");
                return !isReregisterMessage;
            }
            
        });

**Comment by bgroenks96** [comment]
Yes, you're right. Logging level usage is pretty inconsistent. I'll take a look at what can be done in a reasonable amount of time as soon as possible.
